README.md

```diff
@@ -143,51 +157,66 @@ eclipse does not support hierarchical projects.
-   Detecting the current branch failed: 'git' is not recognized as an internal or external command,
-   operable program or batch file.`
-   +  `git` might not be installed on your system. Make sure that you can execute
+-   `[WARNING] Cannot get the branch information from the git repository:
+     Detecting the current branch failed: 'git' is not recognized as an internal or external command,
+     operable program or batch file.`
+    + `git` might not be installed on your system. Make sure that you can execute
       `git` from the command line.
 
-- `1) Error injecting constructor, java.lang.Error: Unresolved compilation problems:
-        The import org.kframework.parser.outer.Outer cannot be resolved
-        Outer cannot be resolved`
-   + You may run into this issue if target/generated-sources/javacc is not added to the
-     build path of your IDE. Generally this is solved by regenerating your project /
-     re-syncing it with the pom.xml.
-
-- `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile
-   (default-compile) on project k-core: Fatal error compiling: invalid target release: 1.8 -> [Help 1]`
-   + You either do not have Java 8 installed, or `$JAVA_HOME` does not point to a Java 8 JDK.
+-   `1) Error injecting constructor, java.lang.Error: Unresolved compilation problems:
+          The import org.kframework.parser.outer.Outer cannot be resolved
+          Outer cannot be resolved`
+    + You may run into this issue if target/generated-sources/javacc is not added to the
+      build path of your IDE. Generally this is solved by regenerating your project /
+      re-syncing it with the pom.xml.
+
+-   `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile
+     (default-compile) on project k-core: Fatal error compiling: invalid target release: 1.8 -> [Help 1]`
+    + You either do not have Java 8 installed, or `$JAVA_HOME` does not point to a Java 8 JDK.
+
+-   `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.7:run
+     (build-haskell) on project haskell-backend: An Ant BuildException has occured: exec returned: 1`
+
+    and scrolling up, you see an error message similar to:
+
+    `[exec] Installing GHC ...
+     [exec] ghc-pkg: Couldn't open database $HOME/.stack/programs/x86_64-linux/ghc-tinfo6-8.10.1/lib/ghc-8.10.1/package.conf.d for modification:
+     {handle: $HOME/.stack/programs/x86_64-linux/ghc-tinfo6-8.10.1/lib/ghc-8.10.1/package.conf.d/package.cache.lock}:
+     hLock: invalid argument (Invalid argument)`
+    + If you are using a [WSL version 1 environment](https://docs.microsoft.com/en-us/windows/wsl/compare-versions),
+      then you have encountered a known issue with the latest versions of GHC. In this
+      case, please either build the K Framework in a different environment
+      or else install a [packaged release for your WSL version 1 distribution](https://github.com/kframework/k/releases/).
```

Comment:
Using WSL 2 is also an option, and probably a better one; at some point WSL 1 won't be supported any more.

Reply:
@ttuegel I added text indicating this possibility as well as just doing `mvn package -Dhaskell.backend.skip`